### PR TITLE
Handle renaming of sections to tabs (Statamic 4)

### PR DESCRIPTION
**Pull request by justkidding96**

### CODE REVIEW

**Comment by mkwia** [comment]
Addresses #159. Need to test and tag new release for v4.

**Comment by mkwia** [comment]
Thank you for your PR @justkidding96 . The team will review this shortly

**Comment by geertjanknapen1** [comment]
Any updates?

File: src/Listeners/AppendEntrySeoFieldsListener.php [hunk 1]
```diff
@@ -27,9 +27,10 @@ public function handle(EntryBlueprintFound $event)
             $contents = $bp->contents();
 
             $on_page_bp = OnPageSeoBlueprint::requestBlueprint();
-            $on_page_fields = $on_page_bp->contents()['sections']['main'];
+            $on_page_key = (array_key_exists('tabs', $on_page_bp->contents()) ? 'tabs' : 'sections');
+            $on_page_fields = $on_page_bp->contents()[$on_page_key]['main'];
 
-            $contents['sections']['SEO'] = $on_page_fields;
+            $contents[$on_page_key']['SEO'] = $on_page_fields;
```
Comment (geertjanknapen1) [inline src/Listeners/AppendEntrySeoFieldsListener.php:33]: There's a `'` after `$on_page_key` there. That will break.

**Review by geertjanknapen1 — CHANGES_REQUESTED** [review]

File: src/Listeners/AppendTermSeoFieldsListener.php [hunk 2]
```diff
@@ -27,9 +27,10 @@ public function handle(TermBlueprintFound $event)
             $contents = $bp->contents();
 
             $on_page_bp = OnPageSeoBlueprint::requestBlueprint();
-            $on_page_fields = $on_page_bp->contents()['sections']['main'];
+            $on_page_key = (array_key_exists('tabs', $on_page_bp->contents() ? 'tabs' : 'sections');
```
Comment (geertjanknapen1) [inline src/Listeners/AppendTermSeoFieldsListener.php:30]: missing `)` after `(array_key_exists('tabs', $on_page_bp->contents()`

is now;
`$on_page_key = (array_key_exists('tabs', $on_page_bp->contents() ? 'tabs' : 'sections');`
should be;
`$on_page_key = (array_key_exists('tabs', $on_page_bp->contents()) ? 'tabs' : 'sections');`

**Review by geertjanknapen1 — CHANGES_REQUESTED** [review]

**Comment by joshuanoyes** [comment]
Hi @justkidding96, thanks for the PR!

Happy to merge this to acknowledge your contribution but it's possible we'll ultimately go with simply accessing `tabs` rather than `sections` as this is caused by a big renaming effort in Statamic 4.

We're looking to get V4 support out ASAP - I'll confirm later today :+1:

**Comment by joshuanoyes** [comment]
Thanks for the review @geertjanknapen1 :slightly_smiling_face:

**Pull request merged**